roles/core/dns_server/tasks/main.yml

```diff
@@ -112,6 +117,18 @@
   tags:
     - template
 
+- name: template █ Generate override file
+  template:
+    src: override.j2
+    dest: /var/named/override
+    owner: root
+    group: root
+    mode: 0644
+  notify: service █ Restart dns services
+  when: dns_overrides is defined and  zoneconf.changed  # noqa 503
```

Comment:
Small typo here, double space `and  zoneconf`

Reply:
Sorry about that - correction committed.